lib/ui/painting/picture.cc

```diff
@@ -117,8 +118,8 @@ Dart_Handle Picture::RasterizeToImage(sk_sp<SkPicture> picture,
     tonic::DartInvoke(image_callback->Get(), {raw_dart_image});
 
     // image_callback is associated with the Dart isolate and must be deleted
-    // on the UI thread
-    delete image_callback;
+    // on the UI thread.
+    image_callback.reset();
```

Comment:
The comment above this `reset` is a bit ominous.  With shared_ptr it's a bit harder to reason about whether the object is actually deleted here or not.  I'm not sure that it is.